src/umpire/strategy/DynamicPoolMap.cpp

```diff
@@ -35,10 +35,20 @@ DynamicPoolMap::DynamicPoolMap(
       m_next_minimum_pool_allocation_size{
           aligned_round_up(next_minimum_pool_allocation_size)}
 {
+  UMPIRE_LOG(Debug, " { "
```

Comment:
Can you reformat this to look like how the debug constructor log normally looks? I think the format you used before is arg=val for all the arguments.

Reply:
Done